tree.go

```diff
@@ -585,38 +583,21 @@ func (n *InternalNode) FlushAtDepth(depth uint8, flush NodeFlushFn) {
 	}
 }
 
-func (n *InternalNode) Get(k []byte, getter NodeResolverFn) ([]byte, error) {
-	nChild := offset2key(k, n.depth)
+func (n *InternalNode) Get(key []byte, resolver NodeResolverFn) ([]byte, error) {
```

Comment:
Note how `Get(...)` got simplified. We call `GetStem(...)` and return the value at the needed index.
All the special error-values in different branches are respected by `GetStem(...)` now.